tests/scripts/minikube.sh

```diff
@@ -74,7 +74,7 @@ copy_image_to_cluster() {
 }
 
 # configure dind-cluster
-KUBE_VERSION=${KUBE_VERSION:-"v1.7.5"}
+KUBE_VERSION=${KUBE_VERSION:-"v1.8.0"}
```

Comment:
i'm late to this party, but with using 1.8 now in minikube, can we remove the `init_flexvolume()` stuff and just let the rook-agent pod deploy its flexvolume driver as intended?

Reply:
we might still need to launch on an older version for testing on occasion, right? There is a version check for when to restart the kubelet so it seems good to keep it